README.md

```diff
@@ -114,7 +135,12 @@ Your query here
 5. How many offices are in the state of Colorado (United States).
 
 ```
-Your query here
+SELECT count(*) AS list_of_office
+FROM offices                                                             
+WHERE offices.state_id  IN                                              
+    (SELECT states.id
+     FROM states INNER JOIN countries ON countries.id = states.country_id
+     WHERE states.name LIKE 'Colorado%' AND countries.name='United States');
```

Comment:
Given that you used `LIKE` operator, I'd recommend you to use `LIKE '%Colorado%'` to make a more general match.